frontend/src/scenes/session-recordings/player/snapshot-processing/chrome-extension-stripping.ts

```diff
@@ -8,7 +8,13 @@ import { RecordingSnapshot } from '~/types'
 // we have seen some chrome extensions
 // that break playback of session recordings
 // let's try to strip them out
-const CHROME_EXTENSION_DENY_LIST = ['dji-sru']
+const CHROME_EXTENSION_DENY_LIST: Record<string, string> = {
+    'dji-sru': 'snap and read',
```

Comment:
this way we can track what extensions the magic keys represent - and report on them